src/www/vpn_ipsec_pubkeys_edit.php

```diff
@@ -0,0 +1,240 @@
+<?php
+
+/*
+ * Copyright (C) 2019 Pascal Mathis <mail@pascalmathis.com>
```

Comment:
Same here

Reply:
Will also add Deciso B.V. and Manuel Kasper here for the reasons mentioned earlier.